perun-openapi/openapi.yml

```
@@ -5235,13 +5317,61 @@ paths:
         default:
           $ref: '#/components/responses/ExceptionResponse'
 
-  #TODO getFacility
+  /urlinjsonout/resourcesManager/getFacility:
```

```suggestion
  /json/resourcesManager/getFacility:
```
When using GET HTTP method, just use the prefix "/json/" instead of "/urlinjsonout/", because GET implies URL parameters only.